### ISSUE TITLE
Include JRuby in the docker image

### ISSUE THREAD

**Issue by pitr-ch** [rake-compiler/rake-compiler-dock#21]
First of all, thanks for the awesome tool!

I maintain [concurrent-ruby](https://github.com/ruby-concurrency/concurrent-ruby) gem and I need to build java extensions as well as C extensions. I would like to run just one task in the docker to produce all the packages I need for a release, however at the moment JRuby is missing and I cannot compile java extensions. Please consider adding it to the image.

**Comment by sodabrew** [comment]
I always build jruby gems on my host system, and have a cheat-sheet of incantations handy. As it sometimes is a pain to get the right JDK set up on new Mac OS releases, just getting it into a docker file at all would be useful.

While there is not much reusability between native code builds for Unix and Windows and Java, abstracting this at the top level with rake commands to make it easier to cut a release would be welcomed imho.

**Comment by pitr-ch** [comment]
Hi thanks for your reply. There are definitely other solutions. The advantage of having it in one image is that all build rake tasks can run in that image easily without running stuff on different machines (including the host). I got it working with a following commands executed in the image before the build task:
```
sudo apt-get update
sudo apt-get install -y openjdk-8-jdk-headless
sudo -u rvm bash -ic 'rvm install jruby-9.1.13.0'
export JRUBY_HOME=/usr/local/rvm/rubies/jruby-9.1.13.0
```
Of course that is downloaded/installed/compiled each time when the gem is build, which is quite non-optimal. Maybe it can help you to see what would be needed to be added to the Dockerfile so it's executed only once during the image build?

**Comment by larskanis** [comment]
This is implemented in rake-compiler-dock-0.7.0. JRuby is shipped by a dedicated JRuby image, so that the MRI image isn't blown up even more. See the [README](https://github.com/rake-compiler/rake-compiler-dock#jruby-support) how it can be used.

I hope that works for you, but if not, don't hesitate to give me a hint.
